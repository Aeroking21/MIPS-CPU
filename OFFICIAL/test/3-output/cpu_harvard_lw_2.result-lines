RESULT = 10203300
