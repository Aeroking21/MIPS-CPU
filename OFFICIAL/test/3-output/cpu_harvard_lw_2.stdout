WARNING: test/ROM.v:23: $readmemh(test/1-binary/lw_2_instructions.mem): Not enough words in the file for the requested range [-1077936128:-1077933128].
WARNING: test/ROM.v:24: $readmemh(pre_reset_v.mem): Not enough words in the file for the requested range [-1077936136:-1077936128].
Instruction = 26
Instruction r = 26
WARNING: rtl/RAM.v:56: $readmemh(test/5-data/data_lw_2.mem): Not enough words in the file for the requested range [256:1000].
VCD info: dumpfile CPU_TB.vcd opened for output.
ROM MODULE RESULTS:
CPU started
RESULT = 10203300
